Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN git reset --hard HEAD && git status
 # used to build protoc-gen-go
 RUN go generate
 RUN dep ensure -v
-RUN gofmt -w .
+# Exclude vendor from gofmt checks.
+RUN mv vendor ../ && gofmt -w . && mv ../vendor .
```

Comment:
el oh el

Reply:
yeah we still have the open PR on veneur about this, but we're really not gonna have a choice about this sooner rather than later, when we get vendor verification into dep.